app/src/main/java/org/p2p/wallet/home/model/TokenConverter.kt

```diff
@@ -54,7 +56,7 @@ object TokenConverter {
             null
         }
         val total = totalLamports.toBigDecimal().divide(tokenMetadata.decimals.toPowerValue())
-        return Token.Active(
+        val token = Token.Active(
```

Comment:
this method should be renamed to fromEntity or something else
it's not used in any of the remote repository